libpod/container_internal_linux.go

```diff
@@ -2225,6 +2249,92 @@ func (c *Container) copyOwnerAndPerms(source, dest string) error {
 	return nil
 }
 
+// harmlessHostPath validates if a host path is harmless and can be modified
+func (c *Container) harmlessHostPath(path string) bool {
```

Comment:
This is fine for now, but this should be moved into Buildah so that it can take advantage of the list.

Reply:
This has been moved to containers/common